src/main/java/uk/gov/hmcts/reform/coh/controller/OnlineHearingController.java

```diff
@@ -137,11 +141,15 @@ public ResponseEntity createOnlineHearing(@RequestBody OnlineHearingRequest body
             onlineHearingPanelMemberService.createOnlineHearing(ohpMember);
         }
 
-        response.setOnlineHearingId(createdOnlineHearing.getOnlineHearingId().toString());
+        response.setOnlineHearingId(onlineHearing.getOnlineHearingId().toString());
+        onlineHearing.setOnlineHearingState(optOnlineHearingState.get());
+        onlineHearing.registerStateChange();
+        onlineHearing = onlineHearingService.createOnlineHearing(onlineHearing);
```

Comment:
No point having the createonlinehearing return anything because the returned object is never used. Just 'onlineHearingService.createOnlineHearing(onlineHearing);' will do